dspace-server-webapp/src/test/java/org/dspace/app/rest/builder/CollectionBuilder.java

```diff
@@ -179,6 +192,28 @@ public void deleteWorkflowGroups(Collection collection) throws Exception {
         indexingService.commit();
     }
 
+    public void deleteDefaultReadGroups(Collection collection) throws Exception {
+
+        try (Context c = new Context()) {
+            c.turnOffAuthorisationSystem();
+
+            Group defaultItemReadGroup = groupService.findByName(c, "COLLECTION_" +
+                collection.getID().toString() + "_ITEM_DEFAULT_READ");
+            Group defaultBitstreamReadGroup = groupService.findByName(c, "COLLECTION_" +
+                collection.getID().toString() + "_BITSTREAM_DEFAULT_READ");
+            if (defaultItemReadGroup != null) {
+                groupService.delete(c, defaultItemReadGroup);
+            }
+            if (defaultBitstreamReadGroup != null) {
+                groupService.delete(c, defaultBitstreamReadGroup);
+            }
+
+            c.complete();
+        }
+
+        indexingService.commit();
```

Comment:
I'm not sure this is needed.  Why do we need to commit to the indexing service here?  We don't seem to need to do this when deleting any other type of Group (see other delete methods in this same Builder).  If it's unnecessary, I'd recommend removing it